### DISCUSSION TITLE
Feature to claim free weekly reward in cs2

### DISCUSSION THREAD

**Discussion by ALPhaHoai** [SteamRE/SteamKit#1456] [General]
### What problem is this feature trying to solve?

 claim free weekly reward in cs2

### How would you like it to be solved?

using k_EMsgGCCStrike15_v2_ClientRedeemFreeReward, CMsgGCCstrike15_v2_ClientRedeemFreeReward

### Have you considered any alternative solutions

can't find one

### Additional Information

is this possible to redeem free reward in cs2 using protobuf

**Comment by FallenBE** [comment]
Hello, did you found solution to do it?

**Comment by anyduck** [comment]
https://github.com/gonid0/CS2_Looter
